src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdUserProfileDataLoadScheduler.java

```diff
@@ -28,15 +27,24 @@
     @Value("${start-route}")
     private String startRoute;
 
+    @Value("${scheduler-name}")
+    private String schedulerName;
+
     @PostConstruct
     public void postConstruct() throws Exception {
         camelContext.start();
-        camelContext.getGlobalOptions().put(MappingConstants.ORCHESTRATED_ROUTE, JUDICIAL_USER_PROFILE_ORCHESTRATION);
+        setGlobalOption();
         parentOrchestrationRoute.startRoute();
     }
 
+    private void setGlobalOption() {
+        camelContext.getGlobalOptions().put(uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.ORCHESTRATED_ROUTE, JUDICIAL_USER_PROFILE_ORCHESTRATION);
```

Comment:
this is same code which is used in both scheduler. Can be put in util class and reused.